cardano-crypto-class/cardano-crypto-class.cabal

```diff
@@ -52,6 +52,7 @@ library
     Cardano.Crypto.Hash.Class
     Cardano.Crypto.Hash.Keccak256
     Cardano.Crypto.Hash.NeverUsed
+    Cardano.Crypto.Hash.RIPEMD160
```

Comment:
Can you also add a changelog entry about this. We'll also need to add this new version with the change on master's changelog, once we release `2.1.5.0` on CHaP

Reply:
Sure thing! Just pushed.

I have this tiny change here for `master` as well: https://github.com/IntersectMBO/cardano-base/compare/ripemd-1

Should I open up a PR now with this cosmetic change?

Reply:
> Should I open up a PR now with this cosmetic change?

Yes please. (except on master it should also be in section `2.1.5.0` instead of `2.2.0.0`)